ethereum_history_api/circuits/lib/src/misc/bounded_vecs.nr

```diff
@@ -0,0 +1,7 @@
+pub fn bounded_vec_from_array<T, N, M>(array: [T; N]) -> BoundedVec<T, M> {
+    assert(N <= M, "Array size exceeds bounded vector size");
+    let mut vec = BoundedVec::new();
+    vec.extend_from_array(array);
+
+    vec
+}
```

Comment:
Also: please add PR to stdlib with `BoundedVec<T, N>::from(arr: [T; N])`

Reply:
@aajj999 I will take care of crating this PR.